versioned_docs/version-v2.1.4/token/token-bridging.md

```
@@ -0,0 +1,137 @@
+  </TabItem>
+  <TabItem value="xhtoh" label="Bridge xHOPR (Gnosis) to HOPR (Ethereum)">
+
+:::info
+
+Before initiating the bridging process from Gnosis to the Ethereum chain, please be aware that you will incur two fees: the first on the Gnosis chain to initiate the bridge, and the second on the Ethereum chain for claiming your tokens on the mainnet.
+
+**Important**: Once the bridging process is initiated, it cannot be undone. You must complete the entire bridging process to retrieve your tokens on the Ethereum chain.
+
+:::
+
+## 1. Navigate to the bridge
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)